CMakeLists.txt

```diff
@@ -123,8 +123,13 @@ if(NOT "${SHIBOKEN6_PYTHON_PACKAGE_PATH}" STREQUAL "")
     link_directories("${SHIBOKEN6_PYTHON_PACKAGE_PATH}")
 endif()
 
-file(GLOB PYSIDE6_LIBFILE LIST_DIRECTORIES false "${PYSIDE6_PYTHON_PACKAGE_PATH}/libpyside6.*")
-file(GLOB SHIBOKEN6_LIBFILE LIST_DIRECTORIES false "${SHIBOKEN6_PYTHON_PACKAGE_PATH}/libshiboken6.*")
+if(MSVC)
```

Comment:
We can use a comment for why we need these variable overrides on windows.  If there are some references (URLs) it's good to add them in the comments.  It will greatly help future maintainers.

Reply:
Oh I found this different by browse pyside6 and shiboken6 folder.
I will add a comment here!